lib/Minify/HTML.php

```diff
@@ -134,7 +134,7 @@ public function process()
         $this->_html = preg_replace('/\\s+(<\\/?(?:area|article|aside|base(?:font)?|blockquote|body'
             .'|canvas|caption|center|col(?:group)?|dd|dir|div|dl|dt|fieldset|figcaption|figure|footer|form'
             .'|frame(?:set)?|h[1-6]|head|header|hgroup|hr|html|legend|li|link|main|map|menu|meta|nav'
-            .'|ol|opt(?:group|ion)|output|p|param|section|t(?:able|body|head|d|h||r|foot|itle)'
+            .'|ol|opt(?:group|ion)|output|p|param|section|t(?:able|body|head|d|h||r|foot|title)'
```

Comment:
Thanks for checking, but you missed the preceding `t(?:`. Shorten this group to `t(?:itle)` and you'll see how this works.

Reply:
Ahh, I see. =) Anyway, a bit off topic but I just released an own html minifier. https://github.com/jenstornell/tiny-html-minifier

Reply:
not certainly off-topic. if it's good/maintained/testable library, minify could switch to it.

unfortunately PHP7+ limits such usage for now. minify works with php 5.3, i personally rely on that.

Reply:
@glensc I intend to keep it maintained.

I set PHP7+ to not disapoint anyone, but I've tested it now with https://3v4l.org/ and it works down to 5.4 which I think is fine.